pkg/plugin/plugin.go

```diff
@@ -247,10 +255,6 @@ func (p *plugin) createOrUpdateExt(ctx context.Context, cs *api.OpenShiftManaged
 		return &api.PluginError{Err: err, Step: api.PluginStepGenerateARM}
 	}
 
-	// set VnetID based on VnetName, do this before writing the blobs so that
-	// they are exactly correct
-	cs.Properties.NetworkProfile.VnetID = resourceid.ResourceID(cs.Properties.AzProfile.SubscriptionID, cs.Properties.AzProfile.ResourceGroup, "Microsoft.Network/virtualNetworks", "vnet") // TODO: should be using const
```

Comment:
Can this safely be removed?  Does it cause any problems in the ForceUpdate geneva action?

Reply:
Needs to re-confirm

Reply:
I think this is fine. It moved into `GenerateConfig` method and is set as part of config generation

Reply:
I disagree - GenerateConfig is not run before ForceUpdate?